src/plugin/duration/index.js

```diff
@@ -136,11 +188,11 @@ class Duration {
     if (pUnit === 'milliseconds') {
       base %= 1000
     } else if (pUnit === 'weeks') {
-      base = Math.floor(base / unitToMS[pUnit])
+      base = roundNumber(base / unitToMS[pUnit])
     } else {
       base = this.$d[pUnit]
     }
-    return base
+    return Object.is(base, -0) ? 0 : base
```

Comment:
Do we have to use Object.is here? I think it is not supported on IE

Reply:
updated. use `a === 0` instead